advanced/list-of-log-names.md

```diff
@@ -4,43 +4,43 @@ description: On this page all the available logs are listed
 
 # List of log names
 
-Logs, categorized by what they do are listed here, and are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names(which can be used in place of individual log names, see [configuring channels](../admin-commands/config/configuring-channels.md)) are _Capitalized_. Each one of them is separated by categories in the lists below. If you want to log everything, use an asterisk to include all things that can be logged, including ones that will be introduced in future updates.
+Logs, categorized by what they do are listed here, and are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names (which can be used in place of individual log names, see [_**configuring channels**_](../management-commands/config.md)) are _Capitalized_. Each one of them is separated by categories in the lists below. If you want to log everything, use an asterisk to include all things that can be logged, including ones that will be introduced in future updates.
 
 {% hint style="warning" %}
-Some logs require the bot to have the "View Audit Logs" permission enabled.
+Some logs require the bot to have the "View Audit Log" permission enabled.
 {% endhint %}
 
 ## Moderation
 
-This category is used for actions by moderators or ones that moderators could use. Most of them are fired when a command is run unless they start with AUDIT_. In that case, actions that are done with or without using the bot are logged.
+This category is used for actions by moderators or ones that moderators could use. Most of them are fired when a command is run unless they start with AUDIT_, which uses the values in the server's audit log. In that case, actions that are done with or without using the bot are logged.
```

Comment:
Wording here is convoluted:

1. You don't "use" an action
2. Data is read from the audit log, "uses" is too generic"
3. "with or without" sounds repetitive

Suggested rewrite:
> This category is to log actions taken by moderators. Most of them are triggered by using bot commands, but those starting with \`AUDIT_\` will read the data from Discord's audit log providing MoonlightBot has permission to do so.